src/com/twitter/intellij/pants/service/project/PantsSystemProjectResolver.java

```diff
@@ -82,11 +85,26 @@
     task2executor.put(id, executor);
     final DataNode<ProjectData> projectDataNode =
       resolveProjectInfoImpl(id, executor, listener, isPreviewMode, settings.isEnableIncrementalImport());
+
+    sendSuccessMessage(id);
     doViewSwitch(id, projectPath);
     task2executor.remove(id);
     return projectDataNode;
   }
 
+  private void sendSuccessMessage(@NotNull ExternalSystemTaskId id) {
+    ApplicationManager.getApplication().invokeLater(() -> {
+      Notification myNotification = new Notification(
+        PantsConstants.PANTS,
+        "Success",
+        "Pants project import/refresh succeeded.",
```

Comment:
Should this string live in https://github.com/pantsbuild/intellij-pants-plugin/blob/master/common/com/twitter/intellij/pants/PantsBundle.properties?

Reply:
ah yes. will adjust.